README.rst

```diff
@@ -204,39 +204,13 @@ In order to run the unit tests and to get a coverage report, four additional Pyt
   pip3 install --user pytest
   pip3 install --user coverage
 
-
-Reading the docs
-================
-
-You can read them online at <http://pyccel.readthedocs.io/>.
-
-Alternatively, the documentation can be built automatically using Sphinx.
-First you will need to install a few additional Python packages::
-
-   pip3 install --user sphinx
-   pip3 install --user sphinxcontrib.bibtex
-   pip3 install --user git+git://github.com/saidctb/sphinx-execute-code
-
-Then build the documentation with::
-
-   cd doc
-   make html
-
-Then, direct your browser to ``_build/html/index.html``.
-
 Testing
 =======
 
 To test your Pyccel installation please run the script *tests/run_tests_py3.sh* (Unix), or *tests/run_tests.bat* (Windows).
 
 Continuous testing runs on Travis CI: <https://travis-ci.com/github/pyccel/pyccel>
```

Comment:
@saidctb Since we are updating this file, can you please update this line as well? Our tests do not run on Travis anymore..

Reply:
done

Reply:
Thanks!